build.gradle

```diff
@@ -22,6 +22,7 @@ buildscript {
 		maven { url 'https://repo.spring.io/plugins-release' }
 	}
 	dependencies {
+		classpath("org.springframework.boot:spring-boot-gradle-plugin:1.5.8.RELEASE")
```

Comment:
You should be able to move this dependency declaration to the `spring-credhub-starter` subproject. If that works, you can remove the `springBootVersion` property from this build file.